gremlin-go/driver/connection.go

```diff
@@ -19,50 +19,62 @@ under the License.
 
 package gremlingo
 
+import (
+	"errors"
+	"golang.org/x/text/language"
+)
+
+type connectionState int
+
+const (
+	initialized      connectionState = iota
+	established      connectionState = iota
+	closed           connectionState = iota
+	closedDueToError connectionState = iota
+)
```

Comment:
```suggestion
const (
	initialized      connectionState = iota
	established
	closed
	closedDueToError
)
```
I believe the iota format can be simplified to this